README.md

```diff
@@ -44,7 +44,12 @@ Redis will have:
 
 ### Vault
 
-TBD
+Vault will have:
+
+- a map of Google service account keys at `/credentials/google/${serviceAccountEmail}`
+- a map of AWS IAM keys at `/credentials/amazon/${accountName}`
+- common provider specific data shared amongst clusters at `/clusters/common/${provider}`
+- a map of clusters `/clusters/${channelName}/${clusterName}` containing all sensitive information specific to a single cluster
```

Comment:
Since clusters can be in multiple channels, I've been calling this `environment` rather than `channelName`.  By default, every cluster goes into `/production/`

Reply:
ah that's fair

i was envisioning a separate vault in production vs staging so it's possible that we don't need that path at all. it is convenient for having the `common` location though, so for now i'm going to +1 changing it to `environment` instead of `channelName` and move on